### DISCUSSION TITLE
Develop feature: map in shared album is not showing up

### DISCUSSION THREAD

**Discussion by pizzidellamaremma** [photoprism/photoprism#2857] [Q&A] [answered]
I'm trying to share an album with map.
Images are correctly geotagged but the map is not showing up in the shared album. Am I missing something?

(next images is from my admin panel)
![image](https://user-images.githubusercontent.com/39835713/200082176-b7b7c7ba-63f3-411d-a871-da3be5c01240.png)
![image](https://user-images.githubusercontent.com/39835713/200081986-dd68a7ba-74c1-47bd-8116-b25f9a2e03ee.png)
(next images are from the shared link album)
![image](https://user-images.githubusercontent.com/39835713/200080747-6f214cc8-8881-4031-98a7-835044635732.png)
![image](https://user-images.githubusercontent.com/39835713/200080833-74418297-c026-4d61-85c2-d9e1b206ae2f.png)

I'm using the lastest Photoprism develop version, on Docker.
Leave here an image from that album, could be helpful.
[IMG_6201.zip](https://github.com/photoprism/photoprism/files/9942230/IMG_6201.zip)

**Comment by lastzero** [comment, marked as answer]
It will only be shown when you click on a location label of an individual picture, for example in the card view. Maps are not shown in the main navigation as you need an album context. Also the places feature must be enabled.

**Reply by pizzidellamaremma** [reply]
So you have to open it from the album. It's not very intuitive but it seems to work correctly! Thank you!

**Reply by lastzero** [reply]
It's an extra feature we did do in top of the basic functionality that existed before. Does not mean it stays like this forever.
